src/test/kotlin/ratpack/example/kotlin/SiteSpec.kt

```diff
@@ -0,0 +1,72 @@
+package ratpack.example.kotlin
+
+import com.google.common.io.CharStreams
+import org.jetbrains.spek.api.Spek
+import ratpack.test.MainClassApplicationUnderTest
+import java.io.InputStreamReader
+import kotlin.test.assertEquals
+
+class SiteSpec : Spek({
+
+    val aut = MainClassApplicationUnderTest(Main::class.java)
```

Comment:
Sorry for the radio silence, looks great! It also looks like they've made some progress on the `@closeable` ticket, let's keep a close watch and update when it gets implemented.

Thanks again @geoand